### PR TITLE
added project structure for action servers

### PR DESCRIPTION
**Pull request by camtriggerhappy**

### CODE REVIEW

**Comment by camtriggerhappy** [comment]
@H3nryM I'm generating the project skeleton for the various action servers

File: src/drivetrain/package.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>drivetrain</name>
+  <version>0.0.0</version>
+  <description>TODO: Package description</description>
```
Comment (H3nryM) [inline src/drivetrain/package.xml:6]: do we care about these?

Comment (H3nryM) [reply src/drivetrain/package.xml:6]: @camtriggerhappy

Comment (camtriggerhappy) [reply src/drivetrain/package.xml:6]: I'm inclined to let people fill them out as we produce the code for them

**Review by H3nryM — APPROVED** [review]
LGTM. We'll just try to remember that people should fill out their package.xml information

**Comment by H3nryM** [comment]
Same as before i was letting you merge but it took too long

**Pull request merged**